src/StaticFAQ/index.js

```diff
@@ -37,52 +38,58 @@ type Props = {|
 |};
 
 type State = {|
-  value: string,
+  searchText: string,
 |};
 
 class StaticFAQ extends React.Component<Props, State> {
-  state = {
-    value: '',
-  };
-
+  changeSearchText = (text: string) => {}; // eslint-disable-line no-unused-vars
   handleSearchChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
-    this.setState({ value: e.target.value });
+    this.changeSearchText(e.target.value);
   };
-
   handleCancelSearch = () => {
-    this.setState({ value: '' });
+    this.changeSearchText('');
   };
 
-  renderInput = (isSearching: number) => (
-    <Input
-      value={this.state.value}
-      onChange={this.handleSearchChange}
-      placeholder="What can we help you with?"
-      icon={<Search customColor="#bac7d5" />}
-      onReset={isSearching ? this.handleCancelSearch : undefined}
-      dataCy="input-staticFAQ"
-    />
-  );
+  renderInput = (searchText: string) => {
+    const isSearching = searchText.length;
```

Comment:
I would maybe coerce this into a boolean (`> 0`). Small thing I guess, but one would expect an variable named `isSomething` to be boolean.